docs/parcel-api.md

```diff
@@ -0,0 +1,50 @@
+# Parcel API
+Most parcel methods will be called on the parcel itself, with the excpetion being around mounting.
+
+## mounting
+Both mount methods return a [parcel object](/docs/parcel-api.md#parcel-object). The parcel object contains all additional exposed methods.
```

Comment:
I think we should explain the arguments to the mount methods:

**parcelConfig**: a javascript object that has a bootstrap, mount, unmount, and optionally an update function. These functions are [lifecycle methods]() that must return Promises. Alternatively, you can pass in a loading function (which returns a Promise that resolves with the parcel), which is intended for lazy loading parcels.
**props**: a javascript object of properties to be passed to the parcel. This object **must** have a `domElement` prop, which is the dom node that the parcel will mount into.